frontend/coprs_frontend/run/check_for_anitya_version_updates.py

```diff
@@ -122,6 +122,21 @@ def package_from_source(backend, source_json):
         raise Exception('Unsupported backend {0} passed as command-line argument'.format(args.backend))
 
 
+def is_prerelease(version: str) -> bool:
+    """
+    Detect a pre-release version string.
+    """
+    known_suffixes = [
+        ".dev",
+        ".beta",
+        ".rc",
+        ".alpha",
+    ]
+    for suffix in known_suffixes:
+        if suffix in version:
```

Comment:
I am not sure if you are checking for `suffix in version` instead of `version.endswith(suffix)` intentionally or not. I am fine with both. Just in case you wanted to use `endswith`, it could be a one-liner

```python
return version.endswith(tuple(known_suffixes))
```

Reply:
Yeah...  I updated the code a simple way;  and potential fixes are welcome.

I realized there often are versions like `1.2dev.3`.  This is rather an attempt to have **something**, rather than to find a bulletproof solution.